report_generator/outputs/full_report.md

```diff
@@ -0,0 +1,265 @@
+# Code report
+
+**Repository link:** https://repository.byont/
+
+**Commit number:** j230a-al4uf-zmh2
```

Comment:
Very good to include!

Reply:
Is there somewhere in all the input a file sha we can construct from all the files we constructed? 
That way we have an alternative to see if our review matches code, besides GH. Like directly on etherscan verified files.

Reply:
Sounds indeed like a good option. Will add it to https://github.com/Byont-Ventures/smart-contract-analysis-tools/pull/30 too.